src/rules.ts

```diff
@@ -149,6 +147,12 @@ export const VERSION_FILTER_RULES: FilterRule[] = [
 	{ source: /\(Deluxe Edition\)$/, target: '' },
 	// 6 Foot 7 Foot (Explicit Version)
 	{ source: /[([]Explicit Version[)\]]/i, target: '' },
+	// 6 Foot 7 Foot - Original
+	{ source: /-\sOriginal$/i, target: '' },
+	// California Love - Original Version
+	{ source: /-\sOriginal Version$/i, target: '' },
+	// Personal Jesus - Original Single Version
+	{ source: /-\sOriginal Single Version$/i, target: '' },
```

Comment:
What if we change this rule to remove "Original Whatever Version" ("Whatever" is literally whatever)? E.g. [this album](https://open.spotify.com/album/6LIc6in2ES8hf4ETS8oIyX) contains tracks with such suffix in their names, so the updated rule will also cover this case.